scripts/get-elixir-vsn.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+elixir -e "System.version() |> IO.puts()"
```

Comment:
Does this force everyone to install elixir?

Reply:
Maybe check `command -v elixir` and print “” to avoid error.

Reply:
You are right, since it's in the Makefile.  I'll add the check!  Thanks!